src/components/App.vue

```diff
@@ -42,6 +42,7 @@ export default class App extends ( Vue as VueConstructor<AppBindings> ) {
 		return Promise.all( [
 			store.dispatch( action( NS_LANGUAGE, LANGUAGE_INIT ) ),
 			store.dispatch( action( NS_ENTITY, ENTITY_INIT ), request.entityId ),
+			store.dispatch( action( NS_ENTITY, DETERMINE_EDITABILITY ) ),
```

Comment:
I'd be more comfortable with this being invoked as a consequence of `action( NS_ENTITY, ENTITY_INIT )` - can you relate to that or does it feels like this wires too much use case specific stuff into a store that should work for all use cases? Let's discuss.

Reply:
done